File: Comparativa Python/Ejercicio1-Tablas.py
"""1. Hacer un programa para rellenar una matriz pidiendo al usuario el número
de filas y columnas. Posteriormente, mostrar la matriz en pantalla."""

renglones = int(input('Digite el número de renglones o filas: '))
columnas = int(input('Digite el número de columnas: '))

# Definimos matriz o lista de dos dimensiones y la llenamos de 0s

matriz = [([0]*columnas) for renglones in range(renglones)]

for r in range(0, renglones):
    for c in range(0, columnas):
        matriz[r][c] = (int(input('Digite un número [{}][{}]: '
                        .format(r+1, c+1))))
# Mostrar matriz

for r in range(0, renglones):
    for c in range(0, columnas):
        print(matriz[r][c], ' ', end="")
    print(' ')

File: Matrices o Tablas/Ejercicio1-Tablas.cpp
/*1. Hacer un programa para rellenar una matriz pidiendo al usuario el número de filas y columnas. Posteriormente, mostrar la matriz en pantalla.*/

#include<iostream>
#include<stdlib.h>

using namespace std;

int main(){
    int numeros[100][100], filas, columnas;

    cout<<"Digite el numero de filas: ";
    cin>>filas;
    cout<<"Digite el número de columnas: ";
    cin>>columnas;

    // Almacenando elementos en la matriz
    for (int i = 0; i<filas; i++){
        for (int j = 0; j<columnas; j++){
            cout<<"Digite un numero ["<<i<<"]["<<j<<"]: "; //[0][1]
            cin >>numeros[i][j];
        }
    }

    //Mostrando la matriz
        for (int i = 0; i<filas; i++){
            for (int j = 0; j<columnas; j++){
                cout<<numeros[i][j];
            }
            cout<<endl;
        }    
    system("pause");
    return 0;
}